subsys/bluetooth/mesh/dfu_cli.c

```diff
@@ -464,16 +465,18 @@ static void transfer(struct bt_mesh_blob_cli *b)
 		return;
 	}
 
-	if (is_self_update(cli)) {
-		/* If distributor only updates itself, proceed to the refresh step immediately. */
-		refresh(cli);
-		return;
-	}
-
 	/* If distributor is in the targets list, disable it until Retrieve Capabilities and BLOB
```

Comment:
This comment is now relevant for `bt_mesh_has_addr()` in `skip_targets_from_broadcast`, so I would suggest to move it there.